src/components/Link/Link.stories.tsx

```diff
@@ -12,7 +23,77 @@ type Story = StoryObj<typeof meta>;
 
 export const Default: Story = {
   args: {
-    href: '/test',
-    children: 'Link Text'
+    href: '#',
+    children: <LinkText>Link Text</LinkText>
+  }
+};
+
+export const ListLink: Story = {
+  args: {
+    ...Default.args,
+    type: 'list'
   }
 };
+
+export const Destructive: Story = {
+  args: {
+    ...Default.args,
+    type: 'destructive'
+  }
+};
+
+export const WithIcon: Story = {
+  args: {
+    ...Default.args,
+    hasIcon: true,
+    type: 'list'
+  },
+  render: arguments_ => (
+    <Link {...arguments_}>
+      <LinkText>Document link</LinkText> <Icon name='document' />
+    </Link>
+  )
+};
+
+export const WithIconNoWrapping: Story = {
+  args: {
+    ...WithIcon.args,
+    noWrap: true
+  },
+  render: arguments_ => (
+    <Link {...arguments_}>
+      <LinkText>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Obcaecati
+        incidunt explicabo, odio delectus quia magnam non . teeeeext
+      </LinkText>
+
+      <Icon name='document' />
+    </Link>
+  )
+};
+
+export const JumpLink: Story = {
+  args: {
+    ...Default.args,
+    isJump: true
+  },
+  render: arguments_ => (
+    <Link {...arguments_}>
+      <LinkText>Jump link</LinkText>
+      <Icon name='arrow-right' />
```

Comment:
Want to change these to just left/right instead of the arrows to better match https://cfpb.github.io/design-system/components/links#jump-links-1?

```suggestion
      <Icon name='right' />
```